.github/workflows/build.yml

```
@@ -25,5 +26,15 @@
+        DOCKER_REGISTRY_URL: 'localhost:5000'
+        DOCKER_LOGIN: ""
+        DOCKER_PASSWORD: ""
+        ONEC_USERNAME: ${{ secrets.ONEC_USERNAME}}
+        ONEC_PASSWORD: ${{ secrets.ONEC_PASSWORD}}
+        ONEC_VERSION: '8.3.24.1624'
+        EDT_VERSION: '2023.3.6'
       run: |
-        docker build -t oscript:latest -f ${{ matrix.dir }}/Dockerfile .
+        docker run -d -p 5000:5000 --name registry registry:2
+        ./${{ matrix.script }}
+        
```

yamllint

[error] trailing spaces

(trailing-spaces)